Tools/GNUMake/Make.defs

```diff
@@ -536,6 +542,14 @@ ifeq ($(USE_ACC),TRUE)
 
 endif
 
+ifeq ($(USE_CUPTI),TRUE)
+  ifeq ($(USE_CUDA),TRUE)
+
+  DEFINES += -DAMREX_USE_CUPTI
+
+  endif
+endif
+
 ifeq ($(USE_HIP),TRUE)
```

Comment:
Should add an error if USE_CUDA=FALSE

Reply:
Added error: 'error CUPTI can only be used with USE_CUDA=TRUE'.